docs/src/main/asciidoc/mongodb-panache.adoc

```diff
@@ -87,6 +87,37 @@ If you don't want to generate a new project, add the dependency in your build fi
 implementation("io.quarkus:quarkus-mongodb-panache")
 ----
 
+[NOTE]
+====
+If your project is already configured to use other annotation processors, you will need to additionally add the panache annotation processor:
+
+[source,xml,role="primary asciidoc-tabs-target-sync-cli asciidoc-tabs-target-sync-maven"]
+.pom.xml
+----
+<plugin>
+    <artifactId>maven-compiler-plugin</artifactId>
+    <version>${compiler-plugin.version}</version>
+    <configuration>
+        <parameters>${maven.compiler.parameters}</parameters>
+            <annotationProcessorPaths>
```

Comment:
Maybe add a line with `<!-- Your existing annotation processor... -->` and a blank line to make it clear it's an addition?

Reply:
That's a good idea. I'll add it in once I get to my desk this morning.